examples/fit_ODNP_data.py

```diff
@@ -0,0 +1,105 @@
+"""Fitting ODNP Datasets for Ksigma
+===================================
+The T1(p) and E(p) integrals are pulled from an H5 file and are fit to extract the cross relaxivity of the sample. The R1(p) is first interpolated by fitting the self relaxivity to a polynomial. This fit is then used to fit the linear regime of the enhancement data. The cross relaxivity is then calculated using the output of these fits as well as the sample parameters that are fed to it (e.g., ppt value, and concentration).
```

Comment:
What does "fit the linear regime" mean? Only the linear regime?

Reply:
my attempt at emphasizing that the initial linear regime should have a decent number of points... suggestions for rewording?

Reply:
rephrased

Reply:
You if the inverse of the self relaxivity